src/lib.rs

```diff
@@ -716,6 +731,26 @@ impl Ulid {
         Utc.timestamp(seconds, nanos)
     }
 
+    /// Returns the timestamp of this ULID as a `OffsetDateTime`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use rusty_ulid::Ulid;
+    /// use std::str::FromStr;
+    ///
+    /// let ulid = Ulid::from_str("01CAH7NXGRDJNE9B1NY7PQGYV7")?;
+    /// let datetime = ulid.datetime();
+    ///
+    /// assert_eq!(datetime.to_string(), "2018-04-07 23:39:50.168 +00:00:00");
+    /// # Ok::<(), rusty_ulid::DecodingError>(())
+    /// ```
+    #[cfg(feature = "time")]
+    pub fn datetime(&self) -> OffsetDateTime {
```

Comment:
Similarly, I'd suggest to have two different methods here. One that returns the chrono type (as before) and one that returns the time type. I'm just struggling how to name it... offsetdatetime?

Reply:
offsetdatetime makes sense and keeps it simple, works for me.